pandas/io/formats/latex.py

```diff
@@ -41,6 +41,12 @@ def __init__(
         self.multirow = multirow
         self.clinebuf: List[List[int]] = []
         self.strcols = self._get_strcols()
+
+        # Here is a reason for ignoring typing.
```

Comment:
what is the output of _get_strcols()?

Reply:
``_get_strcols()`` returns List[List[str]], where the inner list contains the elements of the column.